src/languages/php.js

```diff
@@ -56,14 +56,19 @@ export default function(hljs) {
     end: /[ \t]*(\w+)\b/,
     contains: hljs.QUOTE_STRING_MODE.contains.concat(SUBST),
   });
+  const NOWDOC = hljs.END_SAME_AS_BEGIN({
+    begin: /<<<[ \t']*(\w+)'?\n/,
```

Comment:
The `'` here doesn't look correct.  I think you want something like:

```
(\w+|'\w+')
```

What you have now would match `''''''''''WORD` which is not correct.

Reply:
Good catch! Fixed! Made nowdoc more strict. 

Also, quote (") wrapper support added for HEREDOC >>> "END".

PHP allows write `<<<END` OR `<<<"END"` for heredoc. Now `<<<"END"` supported and tests improved.